beamer/state_machine.py

```diff
@@ -438,6 +439,10 @@ def _handle_initiate_l1_resolution(
     if not _timestamp_is_l1_finalized(request.fill_timestamp, context, request.target_chain_id):
         return False, None
 
+    # There is already a process running
+    if (request.id, claim.id) in context.l1_resolutions:
+        return True, None
```

Comment:
I am wondering: why do we have an L1 resolution per `(request, claim)`?
Isn't L1 resolution, in the sense of proving who the filler was, actually per `request`?
The way I see it, once a request is L1 resolved, the result of all existing challenge games becomes known, so there is no need to do multiple L1 resolutions.

Reply:
true.. I don't know why it was like it. Let me think of any edge case, if I can't think of anything, I'll change it.

Reply:
I think it even removes the possibility, that two different claims for the same request would both create a relayer process

Reply:
> I think it even removes the possibility, that two different claims for the same request would both create a relayer process

Exactly.